extension/src/constants/transaction.ts

```diff
@@ -19,3 +23,13 @@ export enum TRANSACTION_WARNING {
   unsafe = "unsafe",
   memoRequired = "memo-required",
 }
+
+export enum CLAIM_PREDICATES {
+  claimPredicateUnconditional = "Unconditional",
+  claimPredicateConditional = "Conditional",
+  claimPredicateAnd = "And",
+  claimPredicateOr = "Or",
+  claimPredicateNot = "Not",
+  claimPredicateBeforeRelativeTime = "Relative Time Before",
+  claimPredicateBeforeAbsoluteTime = "Before Absolute Time",
```

Comment:
💡 The first one should probably lead with before, otherwise it won't make much sense when combined with Not and And.

Reply:
Yes, good thinking. Also, better symmetry with Absolute times